CHANGES.txt

```diff
@@ -5,6 +5,10 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fixed an issue where ``indices.fielddata.breaker`` settings are not
+   changeable during runtime. These settings are marked as deprecated and
+   rewritten to ``indices.breaker.fielddata``.
```

Comment:
i think it is misleading to write "rewritten" here. this could mean rewritten in config etc. 

the point here is that the use is deprecated but still supported as an alias to the new setting in order to not have a breaking change.